common/culture/cultures/wc_amphibian.txt

```diff
@@ -1,19 +1,27 @@
 ﻿murloc = {
 	color = rgb { 217 224 240 }
 	
-	ethos = ethos_communal
+	parents = { gorloc }
+	created = 2.1.1
+
+	ethos = ethos_bellicose
 	heritage = heritage_amphibian
 	language = language_nerglish
 	martial_custom = martial_custom_equal
 	traditions = {
+		tradition_wetlanders
```

Comment:
Imo, I think they should also have storyteller tradition just like gorlocs

](https://wowpedia.fandom.com/wiki/Murloc?so=search#Culture:~:text=Murlocs%20pass%20down%20their%20history%20orally)

Reply:
Do you think it should be added as a sixth tradition or should another tradition should be replaced? Cultures are currently balanced around 5, per the spreadsheet.